package.json

```diff
@@ -8,6 +8,7 @@
     "url": "git@github.com:resin-io/docker-progress.git"
   },
   "author": "Pagan Gazzard <page@resin.io>",
+	"types": "index.d.ts",
```

Comment:
This file seems to be using spaces instead of tabs.

Reply:
God damn it vim...